src/Grpc.AspNetCore.Web/Internal/GrpcWebProtocolConstants.cs

```diff
@@ -0,0 +1,28 @@
+﻿#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+namespace Grpc.AspNetCore.Web.Internal
+{
+    internal static class GrpcWebProtocolConstants
+    {
+        internal const string GrpcContentType = "application/grpc";
```

Comment:
I don't want to because the client, server and web packages are all referenced by functional tests. Sharing a file between all of them will create type name conflicts.

Anyway, it's just a string constant that will never change.